### ISSUE TITLE
Variation between current and earlier versions

### ISSUE THREAD

**Issue by bitsofbits** [GlobalFishingWatch/vessel-classification#180]
* There is significant variation in the classification of vessels since the earlier release.

     - Shows up globally, but no on the test set. Probably because...

     - This is primarily in Chinese vessels, which tend to be in an area of poor coverage
        and we have limited ground truth.

* Two possible explanations: 

    - Change in training data. We added more data since then and the training / test sets were 
       recomputed as a result. 

    - Change in features.

          * Started using cos/sin trick to deal with cyclic parameters (unlikely to matter)

          * Added sin(lat) to so that we can look at seasonal data (could lead to overfitting).

**Comment by bitsofbits** [comment]
David's table of differences:

![image](https://user-images.githubusercontent.com/9038617/32450059-7b97541c-c2d0-11e7-98f8-501836b04baa.png)

**Comment by bitsofbits** [comment]
Experiment 1: 
   * Remove sin(lat), month of year *and* time of day in favor of so called day-angle:
         - 0 at noon, pi/2 at sunset, pi/-pi at midnight and -pi/2 at dawn.
         - linear between sunrise and sunset and sunset and sunrise.

**Comment by bitsofbits** [comment]
Old school features. 

These features are the same as those used for the paper except for bug fixes and using 
sin/cos trick for cyclic parameters. 



run details:

**Comment by bitsofbits** [comment]
Simpler features (branch simpler-features). Reduce features more to make things simpler. Check performance.

     sbt features/'run  --env=dev --zone=us-central1-f --maxNumWorkers=200 --job-name=features-simpler --generate-model-features=True --generate-encounters=False --job-config=feature-pipeline/config/standard_config.yaml --experiments=shuffle_mode=service'
